pandas/plotting/_core.py

```diff
@@ -49,9 +49,13 @@ def _get_standard_kind(kind):
     return {'density': 'kde'}.get(kind, kind)
 
 
-def _gca():
+def _gca(figsize=None):
```

Comment:
Can you change `_gca` to take an optionally dictionary? So like

```python
def _gca(rc=None):
    with plt.rc_context(rc):
        return plt.gca()
```

that way, we aren't limited to just `figsize` when we call gca. You'd call it like `ax = _gca({'figure.figsize': figsize})`